docs/Contributing/release.rst

```diff
@@ -0,0 +1,63 @@
+..
+    SPDX-License-Identifier: GPL-3.0-or-later
+    SPDX-FileCopyrightText: 2022 James Robertson <jwrober@gmail.com>
+
+The Release Process
+*******************
+
+The developers of Freeciv21 release new versions of the software following a cadence. The cadence is loosely
+defined as: 30 non-documentation pull requests (PR's), 200 commits, or 60 days, whichever comes first. We
```

Comment:
```suggestion
defined as: 30 non-documentation pull requests (PRs), 200 commits, or 60 days, whichever comes first. We
```
Plurals are without an apostrophe. Applies to all occurences of "PR's", I just don't want to mark every single one...